frontend/src/MuiForms/CustomTextInput.tsx

```diff
@@ -36,6 +36,7 @@ export default function CustomTextInput(props: CustomTextInputProps) {
     <Fragment key={label}>
       <Typography id={`${id}-label`} fontWeight='bold'>
         {label}
+        {required && <span style={{ color: theme.palette.error.light }}>{' *'}</span>}
```

Comment:
We use `theme.palette.error.main` for all other required asterisks. I'd vote for using the same for all of the ones in this PR too